components/Button.vue

```diff
@@ -5,12 +5,8 @@
         <slot name="prepend" />
       </div>
       <div :class="contentWrapperClasses">
-        {{ $slots.default ? <slot name="default" /> : <slot /> }}
+        {{ $slots.default ? <slot name="default" /> : {{ text }} }}
       </div>
-      <div v-if="!$slots.default" :class="contentWrapperClasses">
-        <slot />
-      </div>
```

Comment:
Vue 不支援像 JSX 用 inline if
```suggestion
      <div v-if="$slots.default" :class="contentWrapperClasses">
        <slot/>
      </div>
      <div v-else :class="contentWrapperClasses">
        {{ text }}
      </div>
```